packages/perseus-editor/src/editor-page.tsx

```diff
@@ -71,7 +71,7 @@ class EditorPage extends React.Component<Props, State> {
     _isMounted: boolean;
     // @ts-expect-error - TS2564 - Property 'rendererMountNode' has no initializer and is not definitely assigned in the constructor.
     rendererMountNode: HTMLDivElement;
-    renderer: RendererInterface | null | undefined;
+    renderer: any;
```

Comment:
This is a bit unfortunate.

Reply:
Yeah, I think it was incorrectly typed though. I don't think it's what we thought it was.